.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        ruby-version: ["2.4", "2.5", "2.6", "3.0"]
+        ruby-version: ["2.6", "3.0"]
```

Comment:
Why this change? That's breaking any existing client on an older version of Ruby, so we'd need a reason, and an entry in the changelog.

Reply:
i need to add into the change log that rdkafka 0.11.0 requires ruby 2.6+ (https://rubygems.org/gems/rdkafka/versions/0.11.0). i was trying to get CI to pass, so i updated that collection, but couldn't find the definition that ensure the project passed on ruby 2.5.

Reply:
:+1: can you also rebase on latest master?